File: check_fb.js
const admin = require('firebase-admin');
const serviceAccount = require('./backend_service_account.json');
admin.initializeApp({
    credential: admin.credential.cert(serviceAccount),
    databaseURL: 'https://riskops-75637-default-rtdb.firebaseio.com'
});
admin.database().ref('shift_reports').limitToLast(10).once('value').then(s => {
    console.log(JSON.stringify(s.val(), null, 2));
    process.exit(0);
});

File: check_fb.py
import firebase_admin
from firebase_admin import credentials, db
import json

cred = credentials.Certificate('./backend_service_account.json')
firebase_admin.initialize_app(cred, {
    'databaseURL': 'https://riskops-75637-default-rtdb.firebaseio.com/'
})

ref = db.reference('shift_reports')
data = ref.order_by_key().limit_to_last(10).get()
print(json.dumps(data, indent=2))
